app/liquid_tags/poll_tag.rb

```diff
@@ -22,98 +22,96 @@ def find_poll(id_code)
+              var html = '<span><span class="ltag-votepercent ltag-'+votedClass+'" style="right:'+percentFromRight+'%"></span>\
+                <span class="ltag-votepercenttext">'+optionText+' — '+roundedPercent+'%</span></span>';
+              pollOptionItem.innerHTML = html;
+              pollOptionItem.classList.add('already-voted')
+              document.getElementById('showmethemoney-'+json.poll_id).innerHTML = '<span class="ltag-voting-results-count">'+totalVotes+' total votes</span>';
+            })
+          }
+           var polls = document.getElementsByClassName('ltag-poll');
+          for (i = 0; i < polls.length; i += 1) {
+            var poll = polls[i]
+            var pollId = poll.dataset.pollId
+            window.fetch('/poll_votes/'+pollId)
+            .then(function(response){
+              response.json().then(
+                function(json){
+                  if (json.voted) {
+                    displayPollResults(json)
+                  } else {
+                    var els = document.getElementById('poll_'+json.poll_id).getElementsByClassName('ltag-polloption')
+                    for (i = 0; i < els.length; i += 1) {
+                      els[i].addEventListener('click', function(e) {
+                        var tokenMeta = document.querySelector("meta[name='csrf-token']")
+                        if (!tokenMeta) {
+                          alert('Whoops. There was an error. Your vote was not counted. Try refreshing the page.')
+                          return
+                        }
+                        var csrfToken = tokenMeta.getAttribute('content')
+                        var optionId = e.target.dataset.optionId
+                        window.fetch('/poll_votes', {
+                          method: 'POST',
+                          headers: {
+                            'X-CSRF-Token': csrfToken,
+                            'Content-Type': 'application/json',
+                          },
+                          body: JSON.stringify({poll_vote: { poll_option_id: optionId } }),
+                          credentials: 'same-origin',
+                        }).then(function(response){
+                          response.json().then(function(j){displayPollResults(j)})
+                        })
+                      });
+                    }
+                    document.getElementById('showmethemoney-'+json.poll_id).addEventListener('click', function(e) {
+                      pollId = this.dataset.pollId
+                      window.fetch('/poll_skips', {
                         method: 'POST',
                         headers: {
                           'X-CSRF-Token': csrfToken,
                           'Content-Type': 'application/json',
                         },
-                        body: JSON.stringify({poll_vote: { poll_option_id: optionId } }),
+                        body: JSON.stringify({poll_skip: {poll_id: pollId }}),
```

Comment:
should this be part of this PR? It seems it's changing behavior here

Reply:
It's `poll_skip` and `poll_id` in master:
https://github.com/thepracticaldev/dev.to/blob/e46bd26223847e35b9de2cd57380bad12a97286f/app/liquid_tags/poll_tag.rb#L96
I wonder why it shows the changes.

Reply:
It's weird, have you tried to rebase?

Reply:
Yep, "up-to-date".

Reply:
It happened to me once or twice, what I did was to create a new branch, cherry pick the commits from the old one in order and either force push or create a new PR.

I still don't know why :(

Reply:
BTW the second commit contains the change so cherry picking it won't work https://github.com/thepracticaldev/dev.to/pull/3255/commits/e2f683752ba3f6ea2fe21e36de142e69ed7f8783

Reply:
Actually, if you look at line 81 red and green - they are identical, and the line 96 red and green - they are identical as well (apart from the indentation). 
But the `git diff` looks confusing.

Reply:
Ah, that's confusing.